src/main/resources/holidays/Holidays_ua.xml

```diff
@@ -32,16 +36,19 @@
 			<tns:MovingCondition substitute="SATURDAY" with="NEXT" weekday="MONDAY"/>
 			<tns:MovingCondition substitute="SUNDAY" with="NEXT" weekday="MONDAY"/>
 		</tns:Fixed>
-		<tns:Fixed month="OCTOBER" day="14" validFrom="2018" descriptionPropertiesKey="DEFENDER_FATHERLAND">
+		<tns:Fixed month="OCTOBER" day="14" descriptionPropertiesKey="DEFENDER_FATHERLAND" validFrom="2015">
```

Comment:
It would be great to introduce a new key, according to the official name of the holiday (since the title used is true only for the Russian Federation). If we want to do everything completely good. ;)

Reply:
I thought that several countries have a similar hollyday and the key is a kind of commonly used. Now I've checked it out and you're right. Before our changes there was only on country using this key and it was Russia. I suggest fixing it in a separate pull-request and combine it with a full ukrainian localization of all supported world-wide holidays.

Reply:
Finally, I've decided to introduce a new resource key instead of reusing existing one. One more commit to this pull request.